### PR TITLE
[Spike] Redo all changes

### PR DESCRIPTION
**Pull request by ken-centrality**
## Summary

After a couple of failed attempts to rebase from upstream, I've decided to redo all the changes that we made so far but will smaller incision cut in hope that it would be easier to merge changes from upstream easier.

A lot of files changed but mostly can be ignored because they are just icons / logos being removed.

- Bring all of our changes into one single package `@trnsp/custom` package
- For MetaMask:
  - Remove the need of `web3-react`
  - Use ReactContext to get MetaMask accounts within very few selective components
  - Can add multiple accounts at once
- Use the new color system to register our chain
- Incorporate the change in #18 

## Checklist

- [x] Add description
- [x] Tag related issue(s)

### CODE REVIEW

**Comment by ken-centrality** [comment]
@aidan-starke @KarishmaBothara please check if I have ported your changes correctly

File: packages/apps-electron/package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
-  "bugs": "https://github.com/polkadot-js/apps/issues",
-  "homepage": "https://github.com/polkadot-js/apps/tree/master/packages/apps-electron#readme",
+  "bugs": "https://github.com/futureversecom/trn-substrate-portal/issues",
+  "homepage": "https://github.com/futureversecom/trn-substrate-portal/tree/master/packages/apps-electron#readme",
   "license": "Apache-2.0",
   "name": "@polkadot/apps-electron",
```
Comment (KarishmaBothara) [inline packages/apps-electron/package.json:5]: if we plan to make it futureverse specific... do we also want to change the name to 
"name": "@futureversecom/apps-electron", in all the package.json?

File: packages/page-accounts/src/Accounts/index.tsx [hunk 2]
```diff
@@ -305,7 +312,7 @@ function Overview ({ className = '', onStatusChange }: Props): React.ReactElemen
           onStatusChange={onStatusChange}
         />
       )}
-      <BannerExtension />
+      {/* <BannerExtension />*/}
```
Comment (KarishmaBothara) [inline packages/page-accounts/src/Accounts/index.tsx:315]: remove the commented code

File: packages/apps-electron/package.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
 {
-  "bugs": "https://github.com/polkadot-js/apps/issues",
-  "homepage": "https://github.com/polkadot-js/apps/tree/master/packages/apps-electron#readme",
+  "bugs": "https://github.com/futureversecom/trn-substrate-portal/issues",
```
[the remark this answers is not in the document]
Comment (ken-centrality) [reply packages/apps-electron/package.json:2]: yeah you are right, I'll change it back :(

File: packages/apps-electron/package.json (hunk 1 above)
Comment (ken-centrality) [reply packages/apps-electron/package.json:5]: ditto, i'll revert this back

File: packages/page-accounts/src/Accounts/index.tsx [hunk 4]
```diff
@@ -26,7 +27,7 @@ import { useTranslation } from '../translate';
 import { SORT_CATEGORY, sortAccounts } from '../util';
 import Account from './Account';
 import BannerClaims from './BannerClaims';
-import BannerExtension from './BannerExtension';
+// import BannerExtension from './BannerExtension';
```
Comment (aidan-starke) [inline packages/page-accounts/src/Accounts/index.tsx:30]:
```suggestion

```

**Review by aidan-starke — APPROVED** [review]
LGTM

**Pull request merged**